tests/cpp_new/ir/ir_builder_test.cpp

```diff
@@ -0,0 +1,22 @@
+#include "gtest/gtest.h"
+
+#include "taichi/ir/ir_builder.h"
+#include "taichi/ir/statements.h"
+
+namespace taichi {
+namespace lang {
+
+TEST(IRBuilder, Basic) {
```

Comment:
googletest is pretty easy to learn: https://google.github.io/googletest/primer.html. By having these files unit tested, we should be able to finalize our IR/passes API, while still iterating and improving the implementations.